sound/soc/intel/boards/Kconfig

```diff
@@ -31,15 +31,29 @@ config SND_SOC_INTEL_HDA_DSP_COMMON
 
 config SND_SOC_INTEL_SOF_MAXIM_COMMON
 	tristate
+	select SND_SOC_MAX98357A
+	select SND_SOC_MAX98373_I2C
+	select SND_SOC_MAX98390
```

Comment:
this is very odd and defeats the purpose of isolating the 'common' parts. I am not following what you are trying to do and why this is necessary. We should only select the minimal configurations.